### ISSUE TITLE
current directory isn't part of an Nx workspace

### ISSUE THREAD

**Issue by birdracoon** [nrwl/nx#12439]
## Current Behavior
When running the tutorial for Angular, the documented way does not work. After creating the workspace with nx serve, the following error message appears: "The current directory isn't part of an Nx workspace.".

## Steps to Reproduce
```
npx create-nx-workspace@latest nxtest2
Need to install the following packages:
  create-nx-workspace@latest
Ok to proceed? (y)
✔ What to create in the new workspace · angular
✔ Application name                    · todo
✔ Default stylesheet format           · scss
✔ Enable distributed caching to make your CI faster · No

 >  NX   Nx is creating your v14.8.3 workspace.

   To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "npm install" several times. Please wait.

✔ Installing dependencies with npm
✔ Nx has successfully created the workspace: nxtest2.
Switched to a new branch 'main'

 >  NX   Successfully initialized git.
``` 

```
❯ cd nxtest2
❯ nx serve todo

>  NX  The current directory isn't part of an Nx workspace.

  To create a workspace run:
  npx create-nx-workspace@latest <workspace name>


>  NX   NOTE  For more information please visit https://nx.dev/

```

#### Root Directory:
```
❯ ls -al
-rwxrwxrwx 1 ----     245 Oct  6 14:02 .editorconfig
-rwxrwxrwx 1 ----     900 Oct  6 14:04 .eslintrc.json
drwxrwxrwx 1 ----     512 Oct  6 14:07 .git
-rwxrwxrwx 1 ----     512 Oct  6 14:04 .gitignore
-rwxrwxrwx 1 ----      74 Oct  6 14:02 .prettierignore
-rwxrwxrwx 1 ----      26 Oct  6 14:02 .prettierrc
drwxrwxrwx 1 ----     512 Oct  6 14:02 .vscode
-rwxrwxrwx 1 ----    3494 Oct  6 14:02 README.md
drwxrwxrwx 1 ----     512 Oct  6 14:04 apps
-rwxrwxrwx 1 ----    2629 Oct  6 14:02 decorate-angular-cli.js
-rwxrwxrwx 1 ----      98 Oct  6 14:04 jest.config.ts
-rwxrwxrwx 1 ----      90 Oct  6 14:04 jest.preset.js
drwxrwxrwx 1 ----     512 Oct  6 14:02 libs
drwxrwxrwx 1 ----     512 Oct  6 14:06 node_modules
-rwxrwxrwx 1 ----    1647 Oct  6 14:04 nx.json
-rwxrwxrwx 1 ---- 1250780 Oct  6 14:05 package-lock.json
-rwxrwxrwx 1 ----    1765 Oct  6 14:04 package.json
drwxrwxrwx 1 ----     512 Oct  6 14:02 tools
-rwxrwxrwx 1 ----     467 Oct  6 14:02 tsconfig.base.json
```

### Environment
I can reproduce this behaviour with Windows, and wsl2 Ubuntu

**Comment by FrozenPandaz** [comment]
I wasn't able to reproduce this issue. Does running `npx nx` solve the issue?

**Comment by birdracoon** [comment]
I have now checked my environment and found that the problem was a very confusing mix in the PATH variable, 
where apparently some wrong node packages were used. i have now cleaned up the environment and now the setup works. 
Thanks for the help

**Comment by fernandobracaroto** [comment]
I'm having this issue. 
When I run npx nx --version I get 15.2.0, but when I run only nx I get The current directory isn't part of an Nx workspace.

**Comment by dupiesdupreez** [comment]
Issue for me too. 

Everything worked up until I switched branches, played around with some newer versions of packages etc, and switched back to my original branch. 

Removed all node_modules etc, but nothing works. Solution or environment is broken now.

@FrozenPandaz Running `npx nx ` did not solve the issue, and running ` nx -v` I get 15.7.0

**Comment by CramericaIndustries** [comment]
I had the same issue after updating to nx `15.7.2`.
Right before the update I did `nx --version` and it told me that I'm on `15.1.1`. Updating went with no errors. After the update completed `nx --version` said `The current directory isn't part of an Nx workspace.`. Doing `npx nx --version` returned `15.7.2` however. 
`npm list -g` showed a slightly old version. So what fixed it for me eventually was uninstalling nx globally and reinstalling it. `npm uninstall -g nx`, `npm i nx -g`. I also rebooted Windows after reinstalling nx.
